### PR TITLE
Actually build OSS app in CI

### PR DESCRIPTION
**Pull request by eseliger**
Setting it to 0 still means "set", so we always built the enterprise app. This fixes it.

### CODE REVIEW

**Comment by felixfbecker** [comment]
🤦🏻 
Should we maybe fix the logic instead that reads it? This mistake could happen locally too.

**Comment by eseliger** [comment]
Added a fix for that, too.

File: client/web/webpack.config.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ const rootDirectory = path.resolve(__dirname, '..', '..')
 const nodeModulesPath = path.resolve(rootDirectory, 'node_modules')
 const monacoEditorPaths = [path.resolve(nodeModulesPath, 'monaco-editor')]
 
-const isEnterpriseBuild = !!process.env.ENTERPRISE
+const isEnterpriseBuild = process.env.ENTERPRISE && Boolean(JSON.parse(process.env.ENTERPRISE))
```
Comment (valerybugakov) [inline client/web/webpack.config.js:30]: Do we need JSON.parse here?

**Review by valerybugakov — APPROVED** [review]
🙌

**Review by limitedmage — APPROVED** [review]

**Pull request merged**